package com.syntaxclass34;

import java.io.IOException;

public class TestException {

public static void main(String[] args)  {
		
	TryCatchFinally.division(10,5);
	TryCatchFinally.division(10, 0);	
	
	System.out.println("-------end of the Code----");
		
			TryCatchFinally.read("");
		
		}

	}
		

	


